plugins/tiddlywiki/filesystem/filesystemadaptor.js

```diff
@@ -122,14 +122,22 @@ FileSystemAdaptor.prototype.loadTiddler = function(title,callback) {
 };
 
 /*
-Delete a tiddler and invoke the callback with (err)
+	Delete a tiddler and invoke the callback with (err)
+	@params title title of tiddler to delete
+	@params callback callback to execute after tiddler deletion
+	@params options `{ cacheOnly: boolean }` cacheOnly: if set cacheOnly as true, we won't touch fs, just delete file from this.boot.files
 */
 FileSystemAdaptor.prototype.deleteTiddler = function(title,callback,options) {
-	var self = this,
-		fileInfo = this.boot.files[title];
+	const fileInfo = this.boot.files[title];
 	// Only delete the tiddler if we have writable information for the file
+	var cacheOnly = options.cacheOnly || false;
+	if(cacheOnly) {
```

Comment:
The "cacheOnly" flag isn't used by the core, so I assume it is intended for plugins? It seems like it would be simpler if it was a separate method

Reply:
Ok, refactor complete!

Reply:
Feel free to point out if you don't like the function name. I think code review is a good place that let people know more about owner's style.